base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -59,7 +59,10 @@ pub struct ConsensusConstants {
     pub(in crate::consensus) emission_tail: MicroTari,
     /// This is the initial min difficulty for the difficulty adjustment
     min_pow_difficulty: (Difficulty, Difficulty),
+    /// The offset relative to the expected genesis coinbase value
+    genesis_coinbase_value_offset: MicroTari,
```

Comment:
This is the faucet?
Should the emmision curve not take care of this?

Reply:
The coinbase in the rincewind genesis block commits to the incorrect value (1000100 uT)